api/src/routing.js

```diff
@@ -146,18 +157,6 @@ app.use(
   })
 );
 
-app.use(function(req, res, next) {
-  var domain = createDomain();
-  domain.on('error', function(err) {
-    logger.error('UNCAUGHT EXCEPTION!');
-    serverUtils.serverError(err, req, res);
```

Comment:
Looks like the old behavior set an error code after an error is hit. Is this still relevant for unhandledRejections where the process doesn't exit?

Reply:
Nice